jenkins_github_checkwarn.sh

```diff
@@ -88,7 +88,25 @@ if [ -n "${GERRIT_PROJECT-}" ]; then
   set -u
 fi
 
-python "${checkpatch_py}"
+env
+pwd
+
+
+set +e
+# Modify the source in place.
+codespell -w --ignore-words-list nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden `git ls-tree --full-tree --name-only HEAD`
```

Comment:
(style)  line over 80 characters
(lint) Use ./*.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden so names with dashes won't become options. [SC2035]
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use $(..) instead of legacy `..`. [SC2006]